packages/react/src/checkbox-group/root/CheckboxGroupRoot.test.tsx

```diff
@@ -13,122 +11,4 @@ describe('<CheckboxGroup.Root />', () => {
     refInstanceof: window.HTMLDivElement,
     render,
   }));
-
-  describe('prop: preserveChildStates', () => {
-    it('should not preserve child states when false', () => {
-      function App() {
-        const [value, setValue] = React.useState<string[]>([]);
-        return (
-          <CheckboxGroup.Root
-            allValues={['red', 'green', 'blue']}
-            value={value}
-            onValueChange={setValue}
-            preserveChildStates={false}
-          >
-            <Checkbox.Root parent data-testid="parent" />
-            <Checkbox.Root name="red" data-testid="red" />
-            <Checkbox.Root name="green" data-testid="green" />
-            <Checkbox.Root name="blue" data-testid="blue" />
-          </CheckboxGroup.Root>
-        );
-      }
-
-      render(<App />);
-
-      const parent = screen.getByTestId('parent');
-      const red = screen.getByTestId('red');
-      const green = screen.getByTestId('green');
-      const blue = screen.getByTestId('blue');
-
-      expect(red).to.have.attribute('aria-checked', 'false');
-      expect(green).to.have.attribute('aria-checked', 'false');
-      expect(blue).to.have.attribute('aria-checked', 'false');
-
-      fireEvent.click(red);
-
-      expect(red).to.have.attribute('aria-checked', 'true');
-      expect(green).to.have.attribute('aria-checked', 'false');
-      expect(blue).to.have.attribute('aria-checked', 'false');
-      expect(parent).to.have.attribute('aria-checked', 'mixed');
-
-      fireEvent.click(parent);
-
-      expect(red).to.have.attribute('aria-checked', 'true');
-      expect(green).to.have.attribute('aria-checked', 'true');
-      expect(blue).to.have.attribute('aria-checked', 'true');
-      expect(parent).to.have.attribute('aria-checked', 'true');
-
-      fireEvent.click(parent);
-
-      expect(red).to.have.attribute('aria-checked', 'false');
-      expect(green).to.have.attribute('aria-checked', 'false');
-      expect(blue).to.have.attribute('aria-checked', 'false');
-      expect(parent).to.have.attribute('aria-checked', 'false');
-
-      fireEvent.click(parent);
-
-      expect(red).to.have.attribute('aria-checked', 'true');
-      expect(green).to.have.attribute('aria-checked', 'true');
-      expect(blue).to.have.attribute('aria-checked', 'true');
-      expect(parent).to.have.attribute('aria-checked', 'true');
-    });
-
-    it('should preserve child states when true', () => {
```

Comment:
Shouldn't we keep this test as a default behavior?

Reply:
It's already tested in `useCheckboxGroupParent.test.tsx`